packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -7,18 +7,19 @@
 pragma solidity ^0.8.18;
 
 import {AddressResolver} from "../../common/AddressResolver.sol";
-import {BlockHeader, LibBlockHeader} from "../../libs/LibBlockHeader.sol";
-import {LibRLPWriter} from "../../thirdparty/LibRLPWriter.sol";
 import {LibTokenomics} from "./LibTokenomics.sol";
 import {LibUtils} from "./LibUtils.sol";
 import {Snippet} from "../../common/IXchainSync.sol";
 import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
 import {TaikoData} from "../../L1/TaikoData.sol";
 
 library LibProving {
-    using LibBlockHeader for BlockHeader;
     using LibUtils for TaikoData.State;
 
+    // keccak256("taiko")
+    bytes32 public constant VERIFIER_OK =
+        0x93ac8fdbfc0b0608f9195474a0dd6242f019f5abc3c4e26ad51fefb059cc0177;
```

Comment:
It should be possible to just do `bytes32 public constant VERIFIER_OK =keccak256("taiko");` and the compiler will calculate it at compile time. Though I remember a couple of years ago it didn't always work reliably but should hopefully be working as expected in all cases.

Reply:
fixed

Reply:
fixed